packages/material-ui-icons/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "material-ui-icons",
-  "private": true,
+  "private": false,
```

Comment:
This is `true` so the build package can't be inadvertently released.

Reply:
It won't work with lerna.